pkgs/tools/misc/wootility/default.nix

```diff
@@ -15,6 +16,20 @@ appimageTools.wrapType2 rec {
     sha256 = "sha256-JodmF3TThPpXXx1eOnYmYAJ4x5Ylcf35bw3R++5/Buk=";
   };
 
+  extraInstallCommands =
+    let contents = appimageTools.extract { inherit pname version src; };
+    in ''
+      source "${makeWrapper}/nix-support/setup-hook"
+      wrapProgram $out/bin/wootility \
+        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
+
+      install -Dm444 ${contents}/wootility-lekker.desktop -t $out/share/applications
+      install -Dm444 ${contents}/wootility-lekker.png -t $out/share/pixmaps
+      substituteInPlace $out/share/applications/wootility-lekker.desktop \
+        --replace-fail 'Exec=AppRun' 'Exec=wootility' \
+        --replace-warn 'Name=wootility-lekker' 'Name=Wootility'
```

Comment:
Why do we replace it?

Reply:
This application is called `Wootility` in all user-facing places (including the nixpkgs pname); the name `wootility-lekker` is an internal codename referring to a specific branch of development (like the `firefox` package is packaging Firefox Quantum; but that's not the user-facing name). See e.g. their website (https://wooting.io/wootility), and names the app uses to reference itself (screenshot attached of titlebar and settings page)

![image](https://github.com/NixOS/nixpkgs/assets/37938646/cf1ccf24-727f-4e06-8c22-751428505fb6)

Additionally, it is stylized with a capital first letter, which is typical for desktop entries, and it matches Wooting's branding (see screenshot, especially titlebar)